src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -103,6 +103,12 @@ public static bool IsPrivilegedProcess
         public static bool IsReleaseRuntime => s_isReleaseRuntime.Value;
         public static bool IsDebugRuntime => s_isDebugRuntime.Value;
 
+        private static readonly Lazy<bool> s_isReleaseLibraries = new Lazy<bool>(() => AssemblyConfigurationEquals("Release"));
+        private static readonly Lazy<bool> s_isDebugLibraries = new Lazy<bool>(() => AssemblyConfigurationEquals("Debug"));
```

Comment:
isn't this the same as s_isReleaseRuntime and s_isDebugRuntime ?

AssemblyConfigurationEquals checks typeof(string).Assembly so I think an easier solution would be to add an overload that takes the assembly.

Reply:
Oh no, those are remnants, they shouldn't be here. The two bellow are important.

Reply:
Removed.

Reply:
ok but why do you need to check DebuggableAttribute instead of adding an overload to the existing AssemblyConfigurationEquals?

Reply:
As far as I can see in my tests, it returns the configuration of runtime, not the library and the attribute is not present for System.Net.Quic.dll:
```
Console.WriteLine(typeof(QuicConnection).Assembly.GetCustomAttribute<AssemblyConfigurationAttribute>() is null); // True
```

Reply:
ah I see, looks like we disable generating `AssemblyConfiguration` attributes for all libraries:
https://github.com/dotnet/runtime/blob/b9acf733a9cc77b02056a2a6d408b6197fb2fe18/eng/versioning.targets#L3-L4